### PR TITLE
#30.confluence.pages

### PR DESCRIPTION
**Pull request by aaschmid**

### CODE REVIEW

File: .env-template [hunk 1]
```diff
@@ -8,12 +8,17 @@ MEETINGS_SHEET_NAME=
 VIRTUAL_OFFICE_BASE_URL=
 VIRTUAL_OFFICE_USERNAME=
 VIRTUAL_OFFICE_PASSWORD=
-GOOGLE_SERVICE_ACCOUNT_MAIL=
-GOOGLE_SERVICE_ACCOUNT_PRIVATE_KEY=
 ENABLE_ROOM_JOIN_MINUTES_BEFORE_START=
 SLACK_BASE_URL=https://<your-domain>.slack.com
 SCHEDULE_DATE=2020-07-03
 SLACK_TOKEN=
+SLACK_ENABLE_RATE_LIMITING=false
```
Comment (klassm) [inline .env-template:15]: Also add it to the readme?

File: src/schedule/ConfluenceClient.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      },
+    };
+  }
+
+  async findLinkForPage(spaceKey: string, title: string): Promise<string | undefined> {
+    const response = await axios.get("/rest/api/content", {
+      ...this.defaultRequestConfig,
+      params: { spaceKey: spaceKey, title: title },
+    });
+    // TODO log
```
Comment (klassm) [inline src/schedule/ConfluenceClient.ts:39]: That TODO logs obsolete

File: .env-template (hunk 1 above)
Comment (aaschmid) [reply .env-template:15]: Ack, done

File: src/schedule/ConfluenceClient.ts (hunk 2 above)
Comment (aaschmid) [reply src/schedule/ConfluenceClient.ts:39]: ack, done

File: src/schedule/createConfluencePages.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+export async function createConfluencePagesAndInsertLinks(office: Office, config: ConfluenceConfig): Promise<Office> {
+  const confluenceClient = new ConfluenceClient(config);
+  const templateStorageContent = await confluenceClient.getPageBody(config.CONFLUENCE_TEMPLATE_PAGE_ID);
+
+  const confluencePages = office.rooms
+    .filter((room) => room.hasConfluencePage)
+    .map((room) => ({ room, pageTitle: `vSR20 - Session ${room.name}` }));
+  for (const { room, pageTitle } of confluencePages) {
+    let pageLink = await confluenceClient.findLinkForPage(config.CONFLUENCE_SPACE_KEY, pageTitle);
```
[the remark this answers is not in the document]
Comment (aaschmid) [reply src/schedule/createConfluencePages.ts:14]: you are right!

**Pull request merged**